index.html

```diff
@@ -0,0 +1,77 @@
+                <input  type="text" name="name" class="form" required placeholder="Full Name"><br>
+                <br>
+                <label><b>Email:</b></label>
+                <input type="email" name="email" class="form" required placeholder="example@mail.com"><br>
+                <br>
+                <label><b>Age:</b></label>
+                <input type="number" name="number" class="form" min="5" max="99" placeholder="12+">
+            </div>
+            <div id="dropdown-div"><br>
+                <label><strong>Which option best describes your current role? </strong></label><br>
+                <select name="role" class="form">
+                    <option value="student">Student</option>
+                    <option value="flJob">Full Time Job</option>
+                    <option value="fllearner">Full Time Learner</option>
+                    <option value="pnt">Prefer Not to Say</option>
+                    <option value="other">Other</option>
+                   
+                </select>
+            </div>
+            <div id="buttons">
+                <br>
+                <label id="recommend"><b>Would you recommend freeCodeCamp to a friend?</b></label><br>
+                <input type="radio" class="radio" name="gender" value="Definetly" checked> Definetly<br>
+                <input type="radio" class="radio" name="gender" value="maybe">Maybe<br>
+                <input type="radio" class="radio" name="gender" value="ns">Not sure<br>
+                <input type="radio" class="radio" name="gender" value="student"> I am a student.(Unemployed)
+            </div>
+          
+    
+            <div id="dropdown"><br><br>
+                <label><strong>What is your favorite feature of freeCodeCamp?</strong></label><br>
+                <select name="feature" class="form">
+                    <option value="frontend">Front-end Projects</option>
+                    <option value="backend">Back-end Projects</option>
+                    <option value="dv">Data Visualization</option>
+                    <option value="challenge">Challenges</option>
+                    <option value="OScommunity">Open Source Community</option>
+                    <option value="Gitter">Gitter help rooms</option>
+                    <option value="Videos">Videos</option>
+                    <option value="Citymeetups">City Meetups</option>
+                    <option value="wiki">Wiki</option>
+                    <option value="forum">Forum</option>
+                    <option value="addCourses">Additional Courses</option>
+                  
+                </select><br><br>
+            </div>
+            
+    
+    
+            <div><br><br>
+                <label>Any comments or suggestions?</label><br>
+                <textarea class="form"
+                    placeholder="Enter your Comment here"></textarea><br><br>
+                <input id="submit" type="submit" class="form">
+            </div>
```

Comment:
```suggestion
    <main id="main">
        <form>
            <section >
                <label><b>Name:</b></label>
                <input  type="text" name="name" class="form" required placeholder="Full Name"><br>
                <br>
                <label><b>Email:</b></label>
                <input type="email" name="email" class="form" required placeholder="example@mail.com"><br>
                <br>
                <label><b>Age:</b></label>
                <input type="number" name="number" class="form" min="5" max="99" placeholder="12+">
            </div>
            <div id="dropdown-div"><br>
                <label><strong>Which option best describes your current role? </strong></label><br>
                <select name="role" class="form">
                    <option value="student">Student</option>
                    <option value="flJob">Full Time Job</option>
                    <option value="fllearner">Full Time Learner</option>
                    <option value="pnt">Prefer Not to Say</option>
                    <option value="other">Other</option>
                   
                </select>
            </div>
            <div id="buttons">
                <br>
                <label id="recommend"><b>Would you recommend freeCodeCamp to a friend?</b></label><br>
                <input type="radio" class="radio" name="gender" value="Definetly" checked> Definetly<br>
                <input type="radio" class="radio" name="gender" value="maybe">Maybe<br>
                <input type="radio" class="radio" name="gender" value="ns">Not sure<br>
                <input type="radio" class="radio" name="gender" value="student"> I am a student.(Unemployed)
            </div>
          
    
            <div id="dropdown"><br><br>
                <label><strong>What is your favorite feature of freeCodeCamp?</strong></label><br>
                <select name="feature" class="form">
                    <option value="frontend">Front-end Projects</option>
                    <option value="backend">Back-end Projects</option>
                    <option value="dv">Data Visualization</option>
                    <option value="challenge">Challenges</option>
                    <option value="OScommunity">Open Source Community</option>
                    <option value="Gitter">Gitter help rooms</option>
                    <option value="Videos">Videos</option>
                    <option value="Citymeetups">City Meetups</option>
                    <option value="wiki">Wiki</option>
                    <option value="forum">Forum</option>
                    <option value="addCourses">Additional Courses</option>
                  
                </select><br><br>
            </div>
            
    
    
            <div><br><br>
                <label>Any comments or suggestions?</label><br>
                <textarea class="form"
                    placeholder="Enter your Comment here"></textarea><br><br>
                <input id="submit" type="submit" class="form">
            </div>
```

Please use HTML 5 semantic tags instead of using many div tags. So that your codes may look more clean.